hm_pyhelper/hardware_definitions.py

```diff
@@ -53,9 +42,11 @@ def is_raspberry_pi() -> bool:
         },
 
     # Nebra Outdoor Hotspot Gen1
-    'NEBHNT-OUT1': {
+    'nebra-outdoor1': {
         'FRIENDLY': 'Nebra Outdoor Hotspot Gen 1',
-        'APPNAME': 'Outdoor',
```

Comment:
@posterzh I have removed appname because I remember there was a discussion about using FRIENDLY in app and this key was not present in all the variants.

Reply:
yep, even the hardware definition key is self-explanatory and become more user-friendly than before.